index.js

```diff
@@ -85,9 +87,15 @@ function executeFlow(_path, options) {
     }
     var result = {};
     result.errors = parsed.errors.filter(function (error) {
+      let lastFile = '';
```

Comment:
'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).